superset/security/manager.py

```diff
@@ -822,6 +806,19 @@ def sync_role_definitions(self) -> None:
         self.get_session.commit()
         self.clean_perms()
 
+    def _get_all_pvms(self) -> list[PermissionView]:
+        # Fetch all pvms
+        pvms = (
+            self.get_session.query(self.permissionview_model)
+            .options(
+                eagerload(self.permissionview_model.permission),
+                eagerload(self.permissionview_model.view_menu),
+            )
+            .all()
+        )
+        pvms = [p for p in pvms if p.permission and p.view_menu]
```

Comment:
pedantic comment but you could just `return` this line instead of updating the var and returning the var.

Reply:
Done, Thanks for the the reviews @giftig .